.github/workflows/release.Hutch.Relay.yml

```diff
@@ -10,30 +10,40 @@ on:
 
 env:
   namespace: hutchstack
-  repo-name: hutch-relay
+  image-name: hutch-relay
+  repo-name: ${{ github.repository }}
+  registry: ghcr.io
 
 jobs:
   publish:
     runs-on: ubuntu-latest
+    permissions:
+      packages: write
     steps:
       - name: Check out the repo
         uses: actions/checkout@v4
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@f95db51fddba0c2d1ec667646a06c2ce06100226 # v3.0.0
+
       - name: Docker Login
-        uses: docker/login-action@v3.1.0
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # v3.0.0
         with:
-          registry: ghcr.io
+          registry: ${{ env.registry }}
           username: ${{github.actor}}
           password: ${{secrets.GITHUB_TOKEN}}
-      
+
       - name: Set timestamp env var
         run: echo "RUN_TIMESTAMP=$(TZ="Etc/UTC" date +'%Y%m%d%H%M%S')" >> $GITHUB_ENV
-      
+
       - name: Docker Metadata action
         id: meta
         uses: docker/metadata-action@v5.5.1
         with:
-          images: ${{ env.namespace }}/${{ env.repo-name }}
+          images: ${{ env.registry }}/${{ env.repo-name }}/${{ env.image-name}}
```

Comment:
I think this is maybe too nested.
(not certain, may depend on repo renaming.)

It currently produces `ghcr.io/health-informatics-uon/rquest-tools/hutch-relay`

I think we want one of the below:
- `ghcr.io/health-informatics-uon/hutch-relay`
- `ghcr.io/health-informatics-uon/hutch/relay`